packages/design-system-scripts/gulp/docs/generatePages/generatePages.js

```diff
@@ -200,32 +200,51 @@ module.exports = async function generatePages(sourceDir, docsDir, options, chang
       processKssSection(kssSection, options)
     )
   );
+  // Get page sections that come from core and child design systems
+  const pageSections = markdownSections.concat(kssSections).map((section) => addCmsdsLink(section));
 
-  // Merge both sets of page sections into a single array.
   // Remove pages with the same URL (so child design systems can override existing pages)
   // Hide sections and pages with the `hide-section` flag
-  const pages = uniquePages(markdownSections.concat(kssSections)).filter(
-    (page) => !page.hideSection
-  );
+  const uniquePageSections = uniquePages(pageSections).filter((page) => !page.hideSection);
 
   // Add react prop and example data to page sections
-  await addReactData(pages);
+  await addReactData(uniquePageSections);
 
-  // Add top-level pages and connect the page sections to their parent pages
-  const nestedPages = await addTopLevelPages(pages).then(nestSections);
+  // Add missing top-level pages and connect the page parts to their parent pages
+  // TODO: remove need to nest pages, or generate from unnested pages
+  const nestedPageSections = await addTopLevelPages(uniquePageSections).then(nestSections);
 
-  // Create HTML files for example and doc pages
   const docsPath = path.join(docsDir, 'dist');
-  const examplePages = await generateExamplePages(pages, docsPath, sourceDir, options, changedPath);
+
+  // Create HTML files for example pages
+  const examplePages = await generateExamplePages(
+    uniquePageSections,
+    docsPath,
+    sourceDir,
+    options,
+    changedPath
+  );
+
   if (changedPath && examplePages > 0) {
     logTask('📝 ', `Example page updated from ${changedPath}`);
   } else if (!changedPath) {
     logTask('📝  ' + examplePages, `Example pages added to ${docsDir}`);
   }
-  const docPages = await generateDocPages(nestedPages, docsPath, options, changedPath);
+
+  // Create HTML files for doc pages
+  const docPages = await generateDocPages(nestedPageSections, docsPath, options, changedPath);
   if (changedPath && docPages > 0) {
     logTask('📝 ', `Doc page updated from ${changedPath}`);
   } else if (!changedPath) {
     logTask('📝  ' + docPages, `Doc pages added to ${docsDir}`);
   }
 };
+function addCmsdsLink(page) {
+  if (
+    page.source.path.includes('node_modules/@cmsgov/design-system-docs/src/pages/components') ||
+    page.source.path.includes('node_modules/@cmsgov/design-system-docs/src/pages/patterns')
+  ) {
```

Comment:
Should we skip the markdown files with file extension `.md`? Can we change the search for core files to components and patterns with file extensions  `.docs.scss` ?
` const regexp = /^node_modules\/.*\/((components|patterns)\/.*\.docs\.scss)$/i;`
`  if (page.source.path.match(regexp)) {`

Reply:
I don't think we want to skip the markdown files here becasue this pattern page would then not be found 
`node_modules/@cmsgov/design-system-docs/src/pages/patterns/documentation-page.md` 

Am i thinking about this correctly?

Reply:
You're right, do we want this pattern page to show the link?

Reply:
Yeah we do want the link to show on the documentation-page